shopelectro/selenium/pages/page.py

```diff
@@ -0,0 +1,20 @@
+from shopelectro.selenium import SiteDriver
+
+
+class Page:
+    """Represent a typical Shopelectro's page.
+
+    Contains cross-page elements: header, footer, ...
+    """
+
+    def __init__(self, driver: SiteDriver):
+        assert isinstance(self.driver, SiteDriver), "Driver must be an instance of shopelectro.selenium.SiteDriver"
+        self.driver = driver
+        self.address: str
+
+    def move_to(self):
```

Comment:
when we read `page.move_to()`, someone can think, that we are moving somewhere outside this page.
I suggest `load` or `open` as name